runtime/src/main/scala/akka/grpc/GrpcChannel.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc
+
+import java.util.concurrent.CompletionStage
+
+import scala.compat.java8.FutureConverters._
+import scala.concurrent.Future
+
+import akka.Done
+import akka.actor.ClassicActorSystemProvider
+import akka.annotation.InternalApi
+import akka.grpc.internal.{ ChannelUtils, InternalChannel }
+import akka.grpc.scaladsl.Grpc
+
+final class GrpcChannel private (val settings: GrpcClientSettings, @InternalApi val internalChannel: InternalChannel)(
```

Comment:
`settings` is used by the generated code, so could you mark it `@InternalStableApi`?

Reply:
Good catch! When I wrote that, I was thinking that it could be convenient for library users if they could access the settings associated with a channel. Looking through the code, however, it looks like settings are generally only consumed and not exposed.

As you suggested, I added the InternalStableApi annotation to settings, and updated internalChannel to use it as well (instead of InternalApi).